st2api/st2api/cmd/api.py

```diff
@@ -74,8 +74,10 @@ def main():
         return _run_server()
     except SystemExit as exit_code:
         sys.exit(exit_code)
-    except Exception:
-        LOG.exception('(PID=%s) ST2 API quit due to exception.', os.getpid())
+    except Exception as e:
+        msg = getattr(e, 'message', str(e))
+        LOG.error('(PID=%s) ST2 API quit due to exception.', os.getpid())
```

Comment:
How come we are changing from LOG.exception to LOG.error?  I thought LOG.exception preserves the original exception in the output log message.